vendor.conf

```diff
@@ -130,7 +130,7 @@ github.com/googleapis/gax-go                        bd5b16380fd03dc758d11cef74ba
 google.golang.org/genproto                          3f1135a288c9a07e340ae8ba4cc6c7065a3160e8
 
 # containerd
-github.com/containerd/containerd                    0edc412565dcc6e3d6125ff9e4b009ad4b89c638 # master (v1.5.0-dev)
+github.com/containerd/containerd                    59a0667cff783b9e4e61ea2a138db220b4fbdca2 # master (v1.5.0-dev)
```

Comment:
I don't see a change in `go.mod` for buildkit, which is still at `0edc412565dc` - perhaps do this separate?